services/net/.devcontainer/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 
 # [Optional] Uncomment this section to install additional OS packages.
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends procps curl ffmpeg libc6-dev libgdiplus
+    && apt-get -y install --no-install-recommends procps curl ffmpeg libc6-dev libgdiplus libgdiplus ffmpeg libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio
```

Comment:
That's a lot of dependencies

Reply:
Not kidding. The documentation assures me they're all required for GStreamer, which allows the processing of mp3 files by Azure.